pricegraph/fuzz/.gitignore

```diff
@@ -0,0 +1,2 @@
+corpus
+artifacts
```

Comment:
```suggestion
artifacts

```

Reply:
I just accepted it for the Cargo.toml but what is the point of newline at end of file? Neither for toml nor for .gitignore does it have any meaning.

Reply:
It’s for version control. Specifically, if you add a new element to the got ignore file, it will show up as 1 line changed and 1 line added. While this is not a huge deal it can interfere with automatic merges (in case, for example, the last element of the got ignore was modified in a separate PR).